go.mod

```diff
@@ -32,8 +31,72 @@ require (
 	k8s.io/klog v1.0.0
 )
 
+require (
```

Comment:
I think we should have a single require section and not two.
@dahuang-purestorage ?
Also, do you see any concerns with these many `require` additions?

Reply:
this is fine. that's a new change in 1.18 where they separate require for direct and indirect dependencies.